src/html/rangeTemplate.html

```diff
@@ -12,11 +12,11 @@
         </div>
         <div class="small-7 medium-6 end columns">
             <div class="row">
-                <input type="button" value="+" class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase" tabindex="1" />
+                <button class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase"><span class="gpii-fd-icon"></span></button>
```

Comment:
I started with plain text, but Dana created an icon that was a bit more stylized, and the "-" was longer.